src/libraries/sendtohelix-wasm.targets

```diff
@@ -12,8 +12,8 @@
   <PropertyGroup Condition="'$(Scenario)' == 'BuildWasmApps'">
     <WorkItemPrefix Condition="'$(TestUsingWorkloads)' == 'true'">Workloads-</WorkItemPrefix>
     <WorkItemPrefix Condition="'$(TestUsingWorkloads)' != 'true'">NoWorkload-</WorkItemPrefix>
-    <WorkItemPrefix Condition="'$(TestUsingWebcil)' == 'false'">$(WorkItemPrefix)NoWebcil-</WorkItemPrefix>
-    <WorkItemPrefix Condition="'$(TestUsingFingerprinting)' == 'false'">$(WorkItemPrefix)NoFingerprint-</WorkItemPrefix>
+    <WorkItemPrefix Condition="'$(WasmEnableWebcil)' == 'false'">$(WorkItemPrefix)NoWebcil-</WorkItemPrefix>
+    <WorkItemPrefix Condition="'$(WasmFingerprintAssets)' == 'false'">$(WorkItemPrefix)NoFingerprint-</WorkItemPrefix>
     <WorkItemPrefix Condition="'$(WasmEnableThreads)' != 'true'">$(WorkItemPrefix)ST-</WorkItemPrefix>
```

Comment:
Please delete these prefixes, they don't make sense in context of WBT

cc @pavelsavara